updates/6/updates.sql

```diff
@@ -0,0 +1,236 @@
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xe11ba472F74869176652C35D30dB89854b5ae84D", 18, "yearn_v2_vault"); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xe11ba472F74869176652C35D30dB89854b5ae84D", "C", "HEGIC yVault", "yvHEGIC", 1610147532, NULL, "", "", "0xe11ba472F74869176652C35D30dB89854b5ae84D"); INSERT INTO underlying_tokens_list(address, weight, parent_token_entry) VALUES ("0x584bC13c7D411c00c01A62e8019472dE68768430", "1.0", "0xe11ba472F74869176652C35D30dB89854b5ae84D");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xE14d13d8B3b85aF791b2AADD661cDBd5E6097Db1", 18, "yearn_v2_vault"); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xE14d13d8B3b85aF791b2AADD661cDBd5E6097Db1", "C", "YFI yVault", "yvYFI", 1615139942, NULL, "", "", "0xE14d13d8B3b85aF791b2AADD661cDBd5E6097Db1"); INSERT INTO underlying_tokens_list(address, weight, parent_token_entry) VALUES ("0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e", "1.0", "0xE14d13d8B3b85aF791b2AADD661cDBd5E6097Db1");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xe2F6b9773BF3A015E2aA70741Bde1498bdB9425b", 6, "yearn_v2_vault"); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xe2F6b9773BF3A015E2aA70741Bde1498bdB9425b", "C", "Yearn USDC Vault", "yvUSDC", 1607188836, NULL, "", "", "0xe2F6b9773BF3A015E2aA70741Bde1498bdB9425b"); INSERT INTO underlying_tokens_list(address, weight, parent_token_entry) VALUES ("0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48", "1.0", "0xe2F6b9773BF3A015E2aA70741Bde1498bdB9425b");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xe9Dc63083c464d6EDcCFf23444fF3CFc6886f6FB", 18, "yearn_v2_vault"); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xe9Dc63083c464d6EDcCFf23444fF3CFc6886f6FB", "C", "Curve oBTC Pool yVault", "yvCurve-oBTC", 1619129210, NULL, "", "", "0xe9Dc63083c464d6EDcCFf23444fF3CFc6886f6FB"); INSERT INTO underlying_tokens_list(address, weight, parent_token_entry) VALUES ("0x2fE94ea3d5d4a175184081439753DE15AeF9d614", "1.0", "0xe9Dc63083c464d6EDcCFf23444fF3CFc6886f6FB");
```

Comment:
Damn these are way too many assets to review manually. How would you suggest I test them?

How did you autogenerate the correct start_timestamp? The others I guess can come from the chain.

Reply:
The timestamp is not correct as I guess there is no way to know it. Does it matter since there is no queries for cc/coingecko?

Reply:
The data comes from calling the curve registry and information on chain so I believe it can be trusted. The SQL queries works fine since I've been using them in my branch

Reply:
> The timestamp is not correct as I guess there is no way to know it. Does it matter since there is no queries for cc/coingecko?

The timestamp is used for log querying. It's important to be correct for ethereum contracts. Since it specifies when the log queries should start from.

Finding the timestamp is also probably possible automatically but it's a bit more complicated if I recall.